codecs/text/src/main/java/com/datastax/oss/dsbulk/codecs/text/string/StringConvertingCodecProvider.java

```diff
@@ -335,11 +335,15 @@ public class StringConvertingCodecProvider implements ConvertingCodecProvider {
               return new StringToDateRangeCodec(nullStrings);
             case DefaultVectorType.VECTOR_CLASS_NAME:
               VectorType vectorType = (VectorType) cqlType;
-              return new StringToVectorCodec(
-                  new VectorCodec(
+              VectorCodec<Number> vectorCodec =
+                  new VectorCodec<>(
                       vectorType,
-                      codecFactory.getCodecRegistry().codecFor(vectorType.getElementType())),
-                  nullStrings);
+                      codecFactory.getCodecRegistry().codecFor(vectorType.getElementType()));
+              ConvertingCodec<JsonNode, List<Number>> jsonCodec =
+                  codecFactory.createConvertingCodec(
+                      DataTypes.listOf(vectorType.getElementType()), JSON_NODE_TYPE, false);
+              return new StringToVectorCodec<>(
+                  vectorCodec, jsonCodec, context.getAttribute(OBJECT_MAPPER), nullStrings);
```

Comment:
See StringToVectorCodec changes below.  jsonCodec is here to convert raw string values into Lists; StringToVectorCodec builds CqlVectors out of them.